docs/stellar-disbursement-platform/getting-started.mdx

```diff
@@ -53,46 +53,55 @@ USDC will arrive in your demo wallet account within seconds! Refresh the account
 
 ## Run the SDP Locally
 
-First you'll need to clone the project.
+### Prerequisites
 
-<CodeExample>
+#### Docker
 
-```bash
-git clone git@github.com:stellar/stellar-disbursement-platform-backend.git
-```
+Make sure you have Docker installed on your system. If not, you can download it from [here](https://docs.docker.com/get-docker/).
 
-</CodeExample>
+#### Hosts
 
-Before we run the SDP, lets replace the private key configured for the distribution account with the one we created with the demo wallet earlier.
+Add the following two hosts to your /etc/hosts file:
 
-<CodeExample>
+<CodeExample>127.0.0.1 sdp-api 127.0.0.1 anchor-platform</CodeExample>
 
-```bash
-cd stellar-disbursement-platform-backend/dev
-```
+#### Stellar Accounts
 
-</CodeExample>
+We will need to create and configure two Stellar accounts to be able to use the SDP.
+
+1. A Distribution account that will be used for sending funds to receivers. [Create and Fund a Distribution Account](https://developers.stellar.org/docs/stellar-disbursement-platform/getting-started#create-and-fund-a-distribution-account)
```

Comment:
I'd add, 'You should have just created this account using the [Create and Fund a Distribution Account] process above.'